File: Stage00-Layout/Python/shared.py
'''
the items, enemies and locations are shared by multiple files, so are best kept in a code module

'''

debug = True
gamestates = {'menu':0, 'play':1, 'dead':2, 'quit':3}
gamestate = gamestates['menu']
items:dict = {}
locations:dict = {}
enemies:dict = {}
current_location:str = ""
	

File: Stage00-Layout/C#/Shared.cs
using System.Collections.Generic;

namespace Adventure_00_Layout
{
    internal static class Shared
    {
        public static bool Debug = true;
        public static Dictionary<string, int> Gamestates = new Dictionary<string, int> 
        {
            { "menu",       0},
            { "play",       1},
            { "leaderboard",2},
            { "quit",       3},
            { "dead",       4}
        };
        public static int Gamestate = 1;
    }
}
